source/_patterns/colors/usage.md

```diff
@@ -0,0 +1,8 @@
+---
+title: Usage
+---
+
+## Accessibility
+
+SVGs are often conveyed inconsistently to assistive technologies. The component’s accessibility is also highly contextual.
+For optimal user experience, use the aria-description prop to let assistive technology users know the purpose of the loading spinner.
```

Comment:
@annsch I currently cannot connect the dots of this information from SVGs to colors.